framework/src/util/LDAPExpr.cpp

```diff
@@ -331,11 +331,6 @@ bool LDAPExpr::IsNull() const
   return !d;
 }
 
-bool LDAPExpr::Query( const std::string& filter, const PropertiesHandle& pd)
```

Comment:
I don't remember, why was this function removed?

Reply:
This function was removed because it's unused in our code (and it's not a public interface)